internal/database/models/models.go

```diff
@@ -2,24 +2,30 @@ package models
 
 import (
 	"gorm.io/gorm"
+	"time"
 )
 
 type Author struct {
-	gorm.Model
-	FirstName     string  `gorm:"not null" json:"first_name"`
-	LastName      string  `json:"last_name"`
-	Bio           string  `gorm:"type:text" json:"bio"`
-	OriginCountry string  `gorm:"not null" json:"origin_country"`
-	CurrentCity   string  `gorm:"not null" json:"current_city"`
-	Latitude      float64 `gorm:"not null" json:"latitude"`
-	Longitude     float64 `gorm:"not null" json:"longitude"`
-	Stories       []Story `gorm:"foreignKey:AuthorID;constraint:OnUpdate:CASCADE,OnDelete:CASCADE;" json:"stories,omitempty"`
+	ID        		uint	`gorm:"autoIncrement:true"`		
```

Comment:
should we get rid of this? it's not being used anywhere right now, and I doubt we will need to query it in the future.